### ISSUE TITLE
THW weiße Schrift und weißer Rand 

### ISSUE THREAD

**Issue by white-gecko** [phjardas/taktische-zeichen#57]
Ähnlich wie #51 wäre es schön statt schwarz, weiß für die Schrift verwenden zu können. Das ist gerade beim THW üblich. Auch den Rand würde ich gerne auf weiß setzen können.

Vielen Dank!

**Comment by phjardas** [comment]
@white-gecko Wenn der Rand weiß ist, soll dann auch die Fachaufgabe in weiß dargestellt werden?

**Comment by white-gecko** [comment]
Ja, ich denke, die Fachaufgabe sollte dann auch weiß sein, wobei ich gerade im THWiki nur Beispiele mit Text finde und Text und Name der Organisation sind ja schon weiß.
